packages/core/src/ceramic.ts

```diff
@@ -184,8 +185,10 @@ const tryStreamId = (id: string): StreamID | null => {
  * To install this library:<br/>
  * `$ npm install --save @ceramicnetwork/core`
  */
-export class Ceramic implements CeramicApi {
-  public readonly context: Context
+export class Ceramic implements StreamReaderWriter, StreamStateLoader {
+  _did?: DID
+  _ipfs?: IpfsApi
+  private _signer: CeramicSigner
```

Comment:
Can we add some comments explaining the relationship between `_did` and `_signer` and when you should use one or the other?